docs/output.rst

```diff
@@ -7,93 +7,96 @@ Outputs of xcp_d
+#.   Functional outputs:
+     Functional outputs consist of processed/denoised BOLD data, timeseries, functional connectivity matrices, and resting-state derivatives.
 
-   a. Denoised or residual BOLD data::
+     #.   Denoised or residual BOLD data::
 
-       # Nifti
-       xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_desc-residual_bold.nii.gz
-       xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_desc-residual_bold.json
+               # Nifti
+               xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_desc-residual_bold.nii.gz
+               xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_desc-residual_bold.json
 
-       # Cifti
-       xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-fsLR_den-91k_desc-residual_bold.dtseries.nii
-       xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-fsLR_den-91k_desc-residual_bold.json
+               # Cifti
+               xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-fsLR_den-91k_desc-residual_bold.dtseries.nii
+               xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-fsLR_den-91k_desc-residual_bold.json
 
-      The json/sidecar contains paramters of the data and processing steps.
+          The json/sidecar contains paramters of the data and processing steps.
 
-       .. code-block:: json-object
+          .. code-block:: json
 
-        {
-              "Freq Band": [ 0.009, 0.08],
-              "RepetitionTime": 2.0,
-              "compression": true,
-              "dummy vols": 0,
-              "nuissance parameters": "27P",
-              }
+               {
+                    "Freq Band": [0.009, 0.08],
+                    "RepetitionTime": 2.0,
+                    "compression": true,
+                    "dummy vols": 0,
+                    "nuisance parameters": "27P",
+               }
 
-   b. Functional timeseries and connectivity matrices::
+     #.   Functional timeseries and connectivity matrices::
 
-        #Nifti
-        xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_atlas-${atlasname}_desc-timeseries_bold.tsv
-        xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_atlas-${atlasname}_desc-connectivity_bold.tsv
+               # Nifti
+               xcp_d/sub-${sub-id}/ses-${ses-id}/func/sub-${sub-id}_ses-${ses-id}_task-${taskname}_run-${run}_space-${space}_atlas-${atlasname}_desc-timeseries_bold.tsv
```

Comment:
Should these be in the conmat format?

Reply:
Good catch. I'll fix that.

Reply:
Oh actually I think I address that in #553.